config/locales/en.yml

```diff
@@ -57,3 +57,11 @@ en:
       full: "%B %-d, %Y"
       youth_date_of_birth: "%B %Y"
       short_date: "%-m/%d"
+  page:
+    #new: I am not sure if the file name should be add?
+      judge: #Should this be Plural as the directory name is plural. Should the
+        title: "New Judge"
+  page:
+    judge:
+      #edit:
+        title: "Edit Judge
```

Comment:
@ErinClaudio thanks for working on this! I would start with defining the active record model and attributes for Judge according to [this documentation](https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models). For example:

```
en:
  activerecord:
    models:
      judge:
        one: Judge
        other: Judges
    attributes:
      judge:
        name: Name
```


Once that is in place, you could create entries for new, edit, delete, etc. like so:

```
en:
  new_record: "New %{model}"
  edit_record: "Edit %{model}"
  destroy_record: "Delete %{model}"
```

And then call it like so:

```
@judge = Judge.first
model_name = @judge.class.model_name.human
new_judge_str = I18n.t(:new_record, model: model_name)
```

Reply:
Thank you for the documentation!